skimage/transform/tests/test_geometric.py

```diff
@@ -747,10 +747,11 @@ def test_inverse_all_transforms(tform):
     assert_almost_equal(tform.inverse.inverse(SRC), tform(SRC))
     # Test addition with inverse, not implemented for all
     if not isinstance(
-        tform,
-        EssentialMatrixTransform
-        | FundamentalMatrixTransform
```

Comment:
man, this syntax is really strange...

Reply:
FYI it seems that this was added by some autofromatting code

https://github.com/scikit-image/scikit-image/commit/d93ce536974116ba8c7f890f6487aedef2bb3b37

@jarrodmillman is there a way for me to disable this kind of syntax reformatting?

Reply:
Yes, I will look into it.
- https://docs.astral.sh/ruff/rules/non-pep604-isinstance/
- https://github.com/astral-sh/ruff/issues/7871

Reply:
See #7430.